packages/sdk-core/src/ConstantFlowAgreementV1.ts

```diff
@@ -237,6 +309,233 @@ export default class ConstantFlowAgreementV1 {
         );
     };
 
+    // CFA ACL Write Functions (byOperator)
```

Comment:
I think if we don't want these title comments to become stale then we need to make them much more visually apparent.

Example:
`/*  ### CFA ACL Write Functions (byOperator) ### */`

Reply:
si